### PR TITLE
Adds isCloudManager check before displaying Product Tour Link

### PR DESCRIPTION
**Pull request by brookescarlett**
## ✍️ Proposed changes

- Adds optional isCloudManager prop to OrgNav
- Adds isCloudManager check before displaying Product Tour Link

🎟 _Jira ticket:_ [Name of ticket](https://jira.mongodb.org/browse/[name-of-ticket])

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

File: packages/mongo-nav/src/org-nav/OrgNav.tsx [hunk 1]
```diff
@@ -139,7 +140,7 @@ function OrgNav({
   admin,
   hosts,
   currentProjectName = 'None',
-
+  isCloudManager = false,
```
Comment (DesignerDave) [inline packages/mongo-nav/src/org-nav/OrgNav.tsx:143]: This will break the current cloud manager implementation, right?

Comment (brookescarlett) [reply packages/mongo-nav/src/org-nav/OrgNav.tsx:143]: No, it shouldn't. OrgNav didn't need to know about whether or not it was on Cloud Manager until now.

Comment (brookescarlett) [reply packages/mongo-nav/src/org-nav/OrgNav.tsx:143]: Only using this boolean to determine whether or not to display the See Product Tour link

Comment (DesignerDave) [reply packages/mongo-nav/src/org-nav/OrgNav.tsx:143]: Sounds good – as long as that prop is added for CM when this version bump happens!

**Pull request closed without merging**